scipy/signal/filter_design.py

```diff
@@ -3653,27 +3653,26 @@ def besselap(N, norm='phase'):
     return asarray([]), asarray(p, dtype=complex), float(k)
 
 
-def iirnotch(w0, bw, rp=3):
+def iirnotch(w0, Q):
     """
     Design second-order IIR notch digital filter.
 
     A notch filter is a band-stop filter with a narrow bandwidth
     (high quality factor). It rejects a narrow frequency band and
     leaves the rest of the spectrum little changed.
 
+    .. versionadded: 0.19.0
+
     Parameters
     ----------
     w0 : float
         Normalized frequency to remove from a signal. It is a
-        scalar that must satisfy  0 < w0 < 1, with w0 = 1 corresponding
-        to half of the sampling frequency.
-    bw : float
-        Normalized filter bandwidth (the diference between up and lower
-        passband edges).
-    rp : float, optional
-        Maximum allowed deviation on the logarithm gain (attenuation)
-        in dB in the passband. By default, `rp` is 3, so the atenuation
-        at the passband edge is -3 dB.
+        scalar that must satisfy  `0 < w0 < 1`, with `w0 = 1`
+        corresponding to half of the sampling frequency.
+    Q : float
+        Quality factor. Dimensionless parameter that characterizes
+        notch filter bandwidth `bw` relative to its center
+        frequency, `Q = w0/bw`.
```

Comment:
That looks good. You need double-backticks, though, since this is meant to be code, not a link.